packages/manager/apps/pci-kubernetes/public/translations/edit-name/Messages_en_GB.json

```diff
@@ -6,5 +6,6 @@
   "pci_projects_project_kubernetes_details_service_name_custom": "Custom name",
   "pci_projects_project_kubernetes_details_service_name_success": "The cluster name has been updated.",
   "pci_projects_project_kubernetes_details_service_name_error": "An error has occurred renaming the cluster: {{message}}}",
-  "pci_projects_project_kubernetes_details_service_name_input_pattern_validation_error": "The name must start with an alphabetical character, end with an alpha-numeric character, and the only special characters allowed are hyphens and underscores."
+  "pci_projects_project_kubernetes_details_service_name_input_pattern_validation_error": "The name must start with an alphabetical character, end with an alpha-numeric character, and the only special characters allowed are hyphens and underscores.",
+  "common_field_error_maxlength": "Please enter a maximum of {{maxlength}} characters."
```

Comment:
**question:** Why common translations are added in the scoped translation files?

Reply:
For the moment this only translation is only used in this modal, but if it's used in other modal / page, we can move it in another namespace